Gaurpal

thakurgaurpal opened the pull request:
following task :"Add side menu on activity task list", have been connected to firebase

thakurgaurpal committed:
updates

thakurgaurpal committed:
Update LoginValidationError.kt

thakurgaurpal committed:
Merge branch 'master' of https://github.com/Gopher-Industries/Guardian

thakurgaurpal committed:
updates for the changes in authentication error message

Here's the code with some changed error message

Co-Authored-By: Lochlan-Ward <104183313+Lochlan-Ward@users.noreply.github.com>

thakurgaurpal committed:
Merge branch 'master' of https://github.com/Gopher-Industries/Guardian

thakurgaurpal committed:
Update androidTestResultsUserPreferences.xml

Co-Authored-By: Lochlan-Ward <104183313+Lochlan-Ward@users.noreply.github.com>

thakurgaurpal committed:
updates on previous toast

Co-Authored-By: Lochlan-Ward <104183313+Lochlan-Ward@users.noreply.github.com>

thakurgaurpal committed:
Task completed

One of the task from trello "Add side menu on activity task list" has been tested and completed.
Thanks

thakurgaurpal committed:
Update deploymentTargetDropDown.xml

thakurgaurpal committed:
add task connected to firebase

waqarsoomro9714 reviewed (changes requested):
please merge only  activity task list,  you  have to work on only activity task list and also when you will create your pull request name should be trello ticket name thank you

thakurgaurpal committed:
week 9 contribution

In this week,
Task "user must not re-use password" is completed
other than this, while working I noticed, it was prompting to login again randomly, so I've fixed this thing as we should not need to login until we logout.
